src/Amazon.Lambda.Tools/Commands/UpdateFunctionConfigCommand.cs

```diff
@@ -247,8 +248,17 @@ protected async Task UpdateConfigAsync(GetFunctionConfigurationResponse existing
                 }
             }
 
-            var urlConfig = await this.GetFunctionUrlConfig(existingConfiguration.FunctionName);
+            GetFunctionUrlConfigResponse urlConfig = null;
 
+            try
+            {
+                urlConfig = await GetFunctionUrlConfig(existingConfiguration.FunctionName);
+            }
+            catch (Exception e)
+            {
+                Logger?.WriteLine($"Exception in [{nameof(GetFunctionUrlConfig)}]: {e.Message}{Environment.NewLine}{e.StackTrace}");
+            }
```

Comment:
Do you know which type of exception is thrown when FunctionURL is not available in the region?

Hiding all the errors will lead to bad experience and non intuitive debugging.